scripts/india_nhm/base/data_cleaner.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+import difflib
+
+INDIA_ISO_CODES = {
+    "Andhra Pradesh": "IN-AP",
+    "Andhra Pradesh Old": "IN-AP",
+    "Arunachal Pradesh": "IN-AR",
+    "Assam": "IN-AS",
+    "Bihar": "IN-BR",
+    "Chattisgarh": "IN-CT",
+    "Chhattisgarh": "IN-CT",
+    "Goa": "IN-GA",
+    "Gujarat": "IN-GJ",
+    "Haryana": "IN-HR",
+    "Himachal Pradesh": "IN-HP",
+    "Jharkhand": "IN-JH",
+    "Jharkhand#": "IN-JH",
+    "Karnataka": "IN-KA",
+    "Kerala": "IN-KL",
+    "Madhya Pradesh": "IN-MP",
+    "Madhya Pradesh#": "IN-MP",
+    "Maharashtra": "IN-MH",
+    "Manipur": "IN-MN",
+    "Meghalaya": "IN-ML",
+    "Mizoram": "IN-MZ",
+    "Nagaland": "IN-NL",
+    "Nagaland#": "IN-NL",
+    "Odisha": "IN-OR",
+    "Punjab": "IN-PB",
+    "Rajasthan": "IN-RJ",
+    "Sikkim": "IN-SK",
+    "Tamil Nadu": "IN-TN",
+    "Tamilnadu": "IN-TN",
+    "Telengana": "IN-TG",
+    "Telangana": "IN-TG",
+    "Tripura": "IN-TR",
+    "Uttarakhand": "IN-UT",
+    "Uttar Pradesh": "IN-UP",
+    "West Bengal": "IN-WB",
+    "Andaman and Nicobar Islands": "IN-AN",
+    "Andaman & Nicobar Islands": "IN-AN",
+    "Andaman & N. Island": "IN-AN",
+    "A & N Islands": "IN-AN",
+    "Chandigarh": "IN-CH",
+    "Dadra and Nagar Haveli": "IN-DN",
+    "Dadra & Nagar Haveli": "IN-DN",
+    "Dadar Nagar Haveli": "IN-DN",
+    "Daman and Diu": "IN-DD",
+    "Daman & Diu": "IN-DD",
+    "Delhi": "IN-DL",
+    "Jammu and Kashmir": "IN-JK",
+    "Jammu & Kashmir": "IN-JK",
+    "Ladakh": "IN-LA",
+    "Lakshadweep": "IN-LD",
+    "Lakshwadeep": "IN-LD",
+    "Pondicherry": "IN-PY",
+    "Puducherry": "IN-PY",
+    "Puduchery": "IN-PY",
+    "Dadra and Nagar Haveli and Daman and Diu": "IN-DN_DD",
+    "all India": "IN",
+    "all-India": "IN",
+    "All India": "IN"
+}
+
+TMCF_ISOCODE = """Node: E:{dataset_name}->E0
+typeOf: schema:Place
+isoCode: C:{dataset_name}->isoCode
```

Comment:
For districts, this might need to be indianCensusAreaCode2011

Reply:
I used lgdCode.